app/assets/stylesheets/application.scss

```diff
@@ -7,3 +7,51 @@
 @import "base/base";
 
 @import "layout";
+
+div.nav {
+
+	ul.menu {
+		@include outer-container(68em);
+		list-style-type: none;
+
+		li.menu-link {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3